cmd/argo/commands/list.go

```diff
@@ -100,6 +114,7 @@ func NewListCommand() *cobra.Command {
 	command.Flags().BoolVar(&listArgs.running, "running", false, "Show only running workflows")
 	command.Flags().StringVarP(&listArgs.output, "output", "o", "", "Output format. One of: wide|name")
 	command.Flags().StringVar(&listArgs.since, "since", "", "Show only workflows newer than a relative duration")
+	command.Flags().Int64VarP(&listArgs.limit, "limit", "", 100, "Limit per num in one request")
```

Comment:
Do you think limit flag should have default 100? All user list will have to make multiple API call. Can we leave it to the user to pass the values once they get an error?

Reply:
you can use the same as kubectl
      --chunk-size=500: Return large lists in chunks rather than all at once. Pass 0 to disable. This flag is beta and
may change in the future.

Reply:
@sarabala1979 Thank you for your review.

Already change code as your comments, pls help review again ,  thanks.